src/main/webapp/app/admin/admin-container/admin-container.component.html

```
@@ -1,7 +1,6 @@
-<div
-    class="flex w-full items-stretch overflow-x-hidden pr-2 h-[calc(100vh_-_var(--sidebar-header-footer-combined-height,105px)_-_15px_-_var(--system-notification-height,0px))]"
-    data-testid="admin-layout"
->
+<!-- Height mirrors the `shell-content-height` mixin the course shells use: everything between the sticky navbar
+     and the footer, minus one grey divider, so the content ends exactly one divider above the footer. -->
+<div class="admin-layout flex w-full items-stretch overflow-x-hidden pr-[var(--spacing-divider)]" data-testid="admin-layout">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)